modules/api/src/main/java/org/eclipse/xpanse/api/controllers/WorkFlowApi.java

```diff
@@ -74,6 +74,18 @@ public List<WorkFlowTask> queryDoneTasks() {
         return workflowUtils.doneTasks(userIdOptional.orElse(null));
     }
 
+    /**
+     * Query all tasks of the given user.
+     */
+    @Tag(name = "Workflow", description = "APIs to manage the Workflow")
+    @Operation(description = "Query all tasks of the given user")
+    @GetMapping(value = "/workflow/tasks", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
we can have URI as /workflow/tasks/all

also looking at these 3 methods, we can also consider combining them into one. /workflow/tasks/${status} where status is optional.